### ISSUE TITLE
Clean Light dont work in nether world

### ISSUE THREAD

**Issue by MrBretze** [bergerhealer/Light-Cleaner#26]
Hello, i use Paper version git-Paper-298 (1.15.2) and my plugin list is: BKCommonLib, Colanta, Graves, LightCleaner, MusicBox, NoteBlockAPI, paperautoupdate, ProtocolLib, Shopkeepers, VoxelSniper, WorldEdit, WorldGuard

I use the last version of BKCommonLib and i dowloaded the last build of LightCleaner on your jenkins

**Comment by bergerkiller** [comment]
Have you downloaded BKCommonLib v4 from jenkins too? You need it for that build of paper. https://ci.mg-dev.eu/job/BKCommonLib/778/

**Comment by MrBretze** [comment]
Yes i have downloaded the jenkins, the V4

**Comment by MrBretze** [comment]
The starting log: https://privatebin.net/?a13fb5c203c0e113#Goc7vCtXpqqKkbPtz894nidFVDq5j5UiqPjyQAW9SQoo

**Comment by bergerkiller** [comment]
Strange, it works in the overworld but in the nether it just hangs. Do you see the same?

**Comment by MrBretze** [comment]
Yes, the light doest update, and i never get the finish message

**Comment by bergerkiller** [comment]
Right, its not even setting new light values, so to me it looks like the chunk loading aspect hangs. I'm looking into it.

**Comment by bergerkiller** [comment]
This one likely needs two updates.

https://ci.mg-dev.eu/job/Light-Cleaner/70/
https://ci.mg-dev.eu/job/BKCommonLib/779/

There was an issue that an error was thrown because light cleaner was trying to set light data for sky light, even though that doesn't exist in the nether. BKC didn't like that.

**Comment by MrBretze** [comment]
Okay i try it

**Comment by MrBretze** [comment]
Its working, just two place have no updated, and i dont have any crash in the console or warning
![2020-05-18_22 08 10](https://user-images.githubusercontent.com/5050734/82255130-693b0680-9954-11ea-9eb9-44f63b647e7d.png)
![2020-05-18_22 08 15](https://user-images.githubusercontent.com/5050734/82255160-78ba4f80-9954-11ea-90a7-75053c7a1f0e.png)

**Comment by bergerkiller** [comment]
does a relogin fix those particular issues?

**Comment by MrBretze** [comment]
No, a relogin dont fix that

**Comment by bergerkiller** [comment]
Then I'm not really sure what to do here, besides just running it a second time to see if anything changes. Could be another light update was queued that ended up overwriting the output of light cleaner. If not, things get hairy, and so far the only way people could help me with bugs like that is to share the world seed and coords where the issue happened.
